CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ FEATURES:
 IMPROVEMENTS:
 
 * resource/vcd_inserted_media now supports force ejecting on running VM [GH-184]
+* resource/vcd_vapp_vm now support cpu cores configuration [GH-168]
```

Comment:
Let's capitalise: cpu -> CPU

Reply:
Wrong GH-168 number? It shows up as "Update repository to use Go modules".

Reply:
Yes, good catch